instaloader/instaloader.py

```diff
@@ -337,6 +337,8 @@ def download_pic(self, filename: str, url: str, mtime: datetime,
         Returns true, if file was actually downloaded, i.e. updated."""
         if filename_suffix is not None:
             filename += '_' + filename_suffix
+        if not url:
+            return False
```

Comment:
I am against adding this, since download_pic should not receive `None` as `url` parameter (and it cannot according to how it is typed).

Reply:
Agreed. It's no longer needed